composer.json

```diff
@@ -168,7 +168,7 @@
             "@test:mutation"
         ],
         "test": [
-            "tools/phpunit/vendor/bin/phpunit"
+            "tools/phpunit/vendor/bin/phpunit --no-coverage"
```

Comment:
This should be passed directly at GitHub actions, I do use coverage from `composer test` locally and by making this change here, it will be no longer available.

Reply:
What about adding: `test:coverage` aside of this?

Reply:
Well technically speaking one of the steps before opening PR should be to check coverage which means that `composer build` will no longer generate coverage, if anything, it could be `test:no-coverage`. 
What's the use case for executing tests without coverage?

Reply:
I don't install xdebug/pcov by default locally as it's slower, thus I always get PHPUnit warning about missing coverage driver.

Anyway reverted & added a new call for usage without coverage that works for me.

Reply:
Hmm right, I think we also need to look into displaying coverage for the new code through GitHub actions, it might be easy to miss that something important wasn't covered when using this option.